presto-docs/src/main/sphinx/optimizer/reorder-joins.rst

```diff
@@ -54,4 +66,5 @@ For the following query:
   SELECT * FROM part p, orders o, lineitem l, supplier s, nation n
   WHERE p.partkey = l.partkey AND l.orderkey = o.orderkey AND l.suppkey = s.suppkey AND s.nationkey = n.nationkey;
```

Comment:
This looks pretty weird with the JOIN syntax because then you need to explicitly say cross join and then have one of the conditions in the where clause, which makes it not a very realistic example.

e.g. `SELECT * FROM part p CROSS JOIN orders o JOIN lineitem l ON l.orderkey = o.orderkey WHERE p.partkey = l.orderkey`